### ISSUE TITLE
PHP 8.3  session_set_save_handler 6 arguments deprecated

### ISSUE THREAD

**Issue by jdarwood007** [SimpleMachines/SMF#7793]
#### Description
The PHP team has deprecated calling session_set_save_handler with 6 or more arguments in 8.3
https://wiki.php.net/rfc/deprecate_functions_with_overloaded_signatures#session_set_save_handler

We call this in 
Session.php
```
			session_set_save_handler('sessionOpen', 'sessionClose', 'sessionRead', 'sessionWrite', 'sessionDestroy', 'sessionGC');
```

The suggestion is to use a object or implant SessionHandlerInterface (PHP 5.4+)

This does affect 2.0 as well

**Comment by sbulen** [comment]
I would recommend this one for 2.1.5.

**Comment by sbulen** [comment]
Actually, I think this has been fixed in 2.1 & 3.0, & may be closed.
